modules/serverless-autoscaling-dashboard-observed-rps.adoc

```diff
@@ -26,5 +26,5 @@ spec:
         autoscaling.knative.dev/metric: "rps"
     spec:
       containers:
-        - image: docker.io/skonto/helloworld-go
+        - image: quay.io/openshift-knative/helloworld-go:metrics
```

Comment:
The image to use here is according to [docs](https://docs.openshift.com/container-platform/4.8/serverless/knative_serving/serverless-applications.html#serverless-about-services_serverless-applications): `docker.io/openshift/hello-openshift`, normal sample app image. The image with the `metrics` tag you references it is just one shot image I pushed for monitoring Knative services.